packages/studio/frontend/src/components/SuperInput/extensions/hoverInspect/index.ts

```diff
@@ -1,4 +1,4 @@
-import { syntaxTree } from '@codemirror/language'
+import { syntaxTree, syntaxParserRunning } from '@codemirror/language'
```

Comment:
Normally, when you save a file where there are unused imports, it will automatically remove the import.